velox/expression/tests/TryExprTest.cpp

```diff
@@ -130,6 +130,37 @@ TEST_F(TryExprTest, nestedTryParentErrors) {
       result);
 }
 
+TEST_F(TryExprTest, skipExecutionEvalSimplified) {
+  registerFunction<CountCallsFunction, int64_t, int64_t>(
+      {"count_calls"}, BIGINT());
+
+  // Test that when a subset of the inputs to a function wrapped in a TRY throw
+  // exceptions, that function is only evaluated on the inputs that did not
+  // throw exceptions.
+  std::vector<std::optional<int64_t>> expected{
```

Comment:
nit: for readability, consider turning expected into a Velox vector and moving it just before it is used:

```
auto expected = makeNullableFlatVector<int64_t>({0, std::nullopt, 1, std::nullopt, 2});
assertEqualVectors(expected, result);
```